packages/web-wallet/src/state/submitDapp/selectors.js

```diff
@@ -0,0 +1,59 @@
+import { createSelector } from 'reselect';
+import { isNil, isEmpty } from 'lodash';
+import { parseBalance, parsedBalanceValue } from '@mobius-network/core';
+
+import { getDappCatalog } from 'state/storeAccount';
+
+export const getSubmitStep = state => state.submitDapp.submitStep;
+
+export const getUserAccount = state => state.submitDapp.userAccount;
+
+export const getUserAccountNumber = state => state.submitDapp.userAccountNumber;
+
+export const getUserAccountId = createSelector(
+  getUserAccount,
+  account => account.id
+);
+
+export const getAsset = (_, { asset = 'mobi' } = {}) => asset;
+
+export const getMobiAsset = () => 'mobi';
```

Comment:
Let's keep prop selectors at the top of the file. All of the selector files are structured like this:

1. Prop selectors and constants
2. Simple selectors
3. Selectors created with `createSelector`